packages/mathtype-tinymce/editor_plugin.src.js

```diff
@@ -286,7 +286,7 @@ export const currentInstance = null;
                   image.removeAttribute('data-mce-style');
                   // Be careful. The next line clear the changes queue
                   // because is only executed these lines on init.
-                  editor.undoManager.clear();
+                  // editor.undoManager.clear();
```

Comment:
I'd rather delete the line if you think it's unnecessary than leave it hanging.

Reply:
okey!